src/e2e/java/uk/gov/hmcts/reform/sscs/functional/ccd/CcdMidEventControllerFunctionalTest.java

```diff
@@ -53,46 +55,53 @@ public void shouldReturn500ForUnauthorised() throws IOException {
                 .statusCode(HttpStatus.SC_INTERNAL_SERVER_ERROR);
     }
 
+    @DisplayName("Should return 200 with valid request")
     @Test
-    public void shouldReturn200WithValidRequest() throws IOException {
+    public void testValidRequest() throws IOException {
         HttpResponse httpResponse = sscsMyaBackendRequests.midEvent(new StringEntity(getJsonCallbackForTest("handlers/writefinaldecision/writeFinalDecisionCallback.json")), "");
-        assertThat(httpResponse.getStatusLine().getStatusCode(), is(200));
+        assertThat(httpResponse.getStatusLine().getStatusCode()).isEqualTo(200);
     }
 
+    @DisplayName("Adjourn case populate venue dropdown should populate next hearing venue dropdown")
     @Test
-    public void adjournCasePopulateVenueDropdown_shouldPopulateNextHearingVenueDropdown() throws IOException {
+    public void testAdjournCasePopulateVenueDropdown() throws IOException {
         HttpResponse httpResponse = sscsMyaBackendRequests.midEvent(new StringEntity(getJsonCallbackForTest("handlers/writefinaldecision/writeFinalDecisionCallback.json")), "AdjournCasePopulateVenueDropdown");
         CcdEventResponse ccdEventResponse = getCcdEventResponse(httpResponse);
-        assertThat(httpResponse.getStatusLine().getStatusCode(), is(200));
+        assertThat(httpResponse.getStatusLine().getStatusCode()).isEqualTo(200);
         DynamicList adjournCaseNextHearingVenueSelected = ccdEventResponse.getData().getAdjournCaseNextHearingVenueSelected();
-        assertThat(adjournCaseNextHearingVenueSelected.getValue(), is(new DynamicListItem("", "")));
-        assertThat(adjournCaseNextHearingVenueSelected.getListItems().size(), is(greaterThan(2)));
+        assertThat(adjournCaseNextHearingVenueSelected.getValue()).isEqualTo(new DynamicListItem("", ""));
```

Comment:
I would split these into two so you know what the problem is if it fails, so:
```suggestion
        assertThat(adjournCaseNextHearingVenueSelected.getValue().getCode()).isEqualTo(""));
        assertThat(adjournCaseNextHearingVenueSelected.getValue().getLabel()).isEqualTo(""));
```

Reply:
done